Makefile

```diff
@@ -0,0 +1,64 @@
+REGISTRY ?= quay.io/kubevirt
+IMAGE_TAG ?= latest
+
+COMPONENTS = $(sort \
+			 $(subst /,-,\
+			 $(patsubst cmd/%/,%,\
+			 $(dir \
+			 $(shell find cmd/ -type f -name '*.go')))))
+
+all: build
+
+build: format $(patsubst %, build-%, $(COMPONENTS))
+
+build-%:
```

Comment:
Are you planning to include more components than just the marker? If not, all these substitution commands `*-%` (`build-%`) are not needed. You can drop them and put the building/testing code directly under `*` (`build`). Instead of `$(subst -,/,$*)` you would use just `marker`.

Reply:
Done